harness/features/variable.local.test.ts

```diff
@@ -74,10 +73,8 @@ describe('Variable Tests - Local', () => {
             // conflicting. This one is used to mock the config that the local client is going to use
             // locally in all of its methods.
             scope
-                .get(
-                    `/client/${testClient.clientId}/config/v1/server/${testClient.sdkKey}.json`,
-                )
-                .reply(200, config, {
+                .get(testClient.getValidConfigPath())
+                .reply(200, testClient.getValidConfig, {
```

Comment:
Don't you need to call the `getValidConfig` method?

Reply:
yep - fixed in the most recent push